swayidle/swayidle.1.txt

```diff
@@ -36,20 +36,19 @@ Options
 *-h*::
 	Show help message and quit.
 
-*-i <idle timeout in seconds>*::
-	Turn off the screen after the system have been idle for the specified 
-	number of seconds.
-
-*-l <lock timeout in seconds>*::
-	Lock the screen after the system have been idle for the specified 
-	number of seconds.
-
 *-x <lock command>*::
-	Raise the lockscreen using the lock command, upon systemd sleep events.
+	Raise the lockscreen using the lock command (by executing it in a shell), upon systemd sleep events.
```

Comment:
Hmm.

Let's get away from this being a flag and get rid of references to the lockscreen. Let's make it behave like timeout/resume:

```
swayidle before-sleep exec 'swaylock...'
```

Reply:
done